index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Personal Portfolio</title>
+  
+  <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+  <header>
+    <nav>
+      <ul class="container">
+        <li class="item"><a href="index.html">Welcome</a></li>
+        <li class="item"><img src="Assets\Images\hamburguer-menu.png"></li>
+      </ul>
+    </nav>
+  </header>
+
+
+
+<section class="first__section">
+  <div class="imgmain"></div>
+  <div class="textmain"><h1>Hey there. I'm Mariane
+    I'm a software developer</h1>
+    <br><br>
+    <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don't hestiate to contact me.</p>
+  </div>
+  <div class="social__media">
+    <div><img src="Assets\Images\Linkedin icon.svg"></div>
```

Comment:
- [ ] Kindly make sure The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.